charts/db-operator/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v2
 type: application
-kubeVersion: ">= 1.19-prerelease <= 1.23-prerelease"
+## All supported k8s versions are in the test: https://github.com/kloeckner-i/db-operator/blob/master/.github/workflows/build-and-test.yaml
+kubeVersion: ">= 1.21"
```

Comment:
we need the `-prerelease` flag, related googles GKE version schema has suffixes and helm then detect an incompatible version if not defined.

Reply:
Ok, got it, thanks, reverted back `-prerelease` flag